shells/android/java/arcs/crdt/parcelables/ParcelableCrdtSingleton.kt

```diff
@@ -0,0 +1,136 @@
+package arcs.crdt.parcelables
+
+import android.os.Parcel
+import android.os.Parcelable
+import arcs.common.Referencable
+import arcs.common.ReferenceId
+import arcs.crdt.CrdtSet
+import arcs.crdt.CrdtSingleton
+
+object ParcelableCrdtSingleton {
```

Comment:
KDoc would be good to have here, and on the public classes who are its members.

Reply:
Done